qa-data/invalid-data-delimiter/data.csv

```diff
@@ -0,0 +1,3 @@
+id,city,,,,,
```

Comment:
@anuveyatsu I thought, if we have extra delimiters in data, it is invalid because of the delimiter. Or I can replace with `;`